GetRecd/Controllers/ProfileSettingsViewController.swift

```diff
@@ -25,10 +23,10 @@ class ProfileSettingsViewController: UITableViewController, UIImagePickerControl
         super.viewDidLoad()
 
 
-        let button = SpotifyLoginButton(viewController: self, scopes: [.streaming, .userLibraryRead])
-        var cell = linkSpotifyButton.superview?.superview!
-        cell?.addSubview(button)
-        button.frame = linkSpotifyButton.frame
+        //let button = SpotifyLoginButton(viewController: self, scopes: [.streaming, .userLibraryRead])
```

Comment:
Should probably delete these commented lines at some point, if the new version we're using works.